packages/flutter/test/widgets/composited_transform_test.dart

```diff
@@ -336,6 +344,100 @@ void main() {
       ),
     );
   });
+  group('FollowerLayerTransformDelegate', () {
+    final GlobalKey key = GlobalKey();
+
+    Widget build(
+        {required Alignment targetAlignment,
+        required Alignment followerAlignment,
+        Offset offset = Offset.zero,
+        required FollowerLayerTransformDelegate delegate}) {
+      return Directionality(
+        textDirection: TextDirection.ltr,
+        child: Stack(
+          children: <Widget>[
+            Positioned(
+              left: 123.0,
+              top: 456.0,
+              child: CompositedTransformTarget(
+                link: link,
+                child: const SizedBox(height: 10.0, width: 10.0),
+              ),
+            ),
+            Positioned(
+              left: 787.0,
+              top: 343.0,
+              child: CompositedTransformFollower(
+                link: link,
+                targetAnchor: targetAlignment,
+                followerAnchor: followerAlignment,
+                offset: offset,
+                transformDelegate: delegate,
+                child: SizedBox(key: key, height: 20.0, width: 20.0),
+              ),
+            ),
+          ],
+        ),
+      );
+    }
+
+    testWidgets('Can override offset with delegate', (WidgetTester tester) async {
+      const Offset testOffset = Offset(100, 200);
+      await tester.pumpWidget(build(
+          targetAlignment: Alignment.topLeft,
+          followerAlignment: Alignment.topLeft,
+          offset: const Offset(10, 20),
+          delegate: TestFixedLocationDelegate(testOffset)));
+      final RenderBox box = key.currentContext!.findRenderObject()! as RenderBox;
+      expect(box.localToGlobal(Offset.zero), const Offset(123, 456) + testOffset);
+    });
+
+    testWidgets('Delegate is passed the correct information', (WidgetTester tester) async {
+      const Offset testOffset = Offset(100, 200);
+      final TestFixedLocationDelegate delegate = TestFixedLocationDelegate(testOffset);
+      await tester.pumpWidget(build(
+          targetAlignment: Alignment.topLeft,
+          followerAlignment: Alignment.topLeft,
+          offset: const Offset(10, 20),
+          delegate: delegate));
```

Comment:
trailing comma formatting nit:

```suggestion
      await tester.pumpWidget(build(
        targetAlignment: Alignment.topLeft,
        followerAlignment: Alignment.topLeft,
        offset: const Offset(10, 20),
        delegate: delegate,
      ));
```